### PR TITLE
Prepare for 0.10.0 release

### PR DESCRIPTION
**Pull request by kiryldz**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,8 +2,9 @@
 
 ## v0.10.0
 
-- Added a new `recorder` module to build `Session Recorder` app
-- Added a new `common` module to share code between `app` and `recorder` modules
+- Added new `recorder` module to build `Session Recorder` app
+- Added new `common` module to share code between `app` and `recorder` modules
+- Added better lifecycle handling of `VisionView`
```
Comment (korshaknn) [inline CHANGELOG.md]:
```suggestion
- Improved `VisionView` lifecycle handling
```

or maybe
- Improved `VisionView` lifecycle management

**Comment by yunikkk** [comment]
@kiryldz we also need to change 
`BUILD_CORE_FROM_SOURCE` checks in gradle files to 
`BUILD_CORE_FROM_SOURCE.toBoolean()`, since otherwise it's treated as string and is always `true`

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,10 +2,20 @@
...
 
+### Technical improvements
+
 - Added new `recorder` module to build `Session Recorder` app
 - Added new `common` module to share code between `app` and `recorder` modules
 - Improved `VisionView` lifecycle management
 
+### User experience
+
+- MapboxVision v0.10.0
```
Comment (yunikkk) [inline CHANGELOG.md]: Don't think it's really worth to mention.
Maybe we can just mention it in `technical` section

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,10 +2,20 @@
...
+- Added detection of construction cones
+- Improved quality of detection/segmentation, especially at night
+- Improved segmentation, now it's more focused on road specific elements. New segmentation model recognizes the following classes: Crosswalk, Hood, MarkupDashed, MarkupDouble, MarkupOther, MarkupSolid, Other, Road, RoadEdge, Sidewalk
+- General bug fixes
```
Comment (yunikkk) [inline CHANGELOG.md]: I'd not add `general` point since there are real changes

**Comment by deniskoronchik** [comment]
Support snap dragon?

File: CHANGELOG.md (hunk 3 above)
Comment (kiryldz) [reply CHANGELOG.md]: Will leave just `Bug fixes` then

File: CHANGELOG.md (hunk 2 above)
Comment (kiryldz) [reply CHANGELOG.md]: Indeed will remove it

**Comment by vladzima** [comment]
What is the reason this was merged without approvals from me and @deniskoronchik?
@yunikkk @kiryldz @korshaknn

**Comment by yunikkk** [comment]
@vladzima we've discussed this changelog in the office, content was identical to iOS Teaser release. 
Since Teaser had to be released on Fri we've merged it right after comments of @deniskoronchik were fixed

**Pull request merged**